esrally/track/params.py

```diff
@@ -515,24 +517,12 @@ def __init__(self, track, params, **kwargs):
             self.query_params["pages"] = pages
         if results_per_page:
             self.query_params["results-per-page"] = results_per_page
+        if "assertions" in params:
+            self.query_params["assertions"] = params["assertions"]
```

Comment:
Should we error if `detailed-results` is False and `assertions` is True?

Reply:
Good idea. :)

Reply:
I've pushed https://github.com/elastic/rally/pull/1157/commits/8286f9fd17eb04666bb370200e81d97f4b924e24 to address your proposal. Can you please have another look @gingerwizard?